components/dictionary.txt

```diff
@@ -1,11 +1,133 @@
+actin
+ADT
+al
 al.
+AML
+AnaLysis
+ASXL
+Azacytidine
+arounds
+Baumer
+Benjamini
+bioconductor
+bioinformatics
+Brems
+CCDL
+CCDL's
+cheatsheet
+cheatsheets
+ChIP
+Cmd
+ColorBrewer
+ComplexHeatmap
+CREB
+Crouser
+cytosolic
+Danio
 dataset
 dataset's
+decitabine
 DESeq
 DESeq2
+DESeqDataset
+devtools
+DGE
+directionality
+DocToc
+Dorsoventral
+ECM
+edgeR
+edgeR's
+eg
```

Comment:
This should always be "e.g." I suspect?
```suggestion
```

Reply:
We can decide in the style guide. In most places we have an italicized `eg` but I am fine with changing it to what we deem appropriate.

Reply:
If we are good grammarians, it should be *e.g.* everywhere, as it is an abbreviation for *exempli gratia*.

Reply:
+1 for "e.g."

Reply:
Addressed in https://github.com/AlexsLemonade/refinebio-examples/pull/257/commits/19a0d74f278ccfd083fd8a1a9065634ab0415f66